File: namsor/kotlin/src/main/kotlin/oseg/namsor_examples/UsRaceEthnicityZIP5Example.kt
package oseg.namsor_examples

import app.namsor.client.infrastructure.*
import app.namsor.client.apis.*
import app.namsor.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class UsRaceEthnicityZIP5Example
{
    fun usRaceEthnicityZIP5()
    {
        ApiClient.apiKey["X-API-KEY"] = "YOUR_API_KEY"

        try
        {
            val response = PersonalApi().usRaceEthnicityZIP5(
                firstName = "Keith",
                lastName = "Haring",
                zip5Code = "12345",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling PersonalApi#usRaceEthnicityZIP5")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling PersonalApi#usRaceEthnicityZIP5")
            e.printStackTrace()
        }
    }
}

File: namsor/typescript-node/src/UsRaceEthnicityZIP5Example.ts
import * as fs from 'fs';
import api from "namsor_client"
import models from "namsor_client"

const apiCaller = new api.PersonalApi();
apiCaller.setApiKey(api.PersonalApiApiKeys.api_key, "YOUR_API_KEY");

apiCaller.usRaceEthnicityZIP5(
  "Keith", // firstName
  "Haring", // lastName
  "12345", // zip5Code
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling PersonalApi#usRaceEthnicityZIP5:");
  console.log(error.body);
});
